mxcube3/core/components/lims.py

```diff
@@ -207,6 +207,12 @@ def lims_login(self, loginID, password, create_session):
                 logging.getLogger("MX3.HWR").error(msg)
                 return ERROR_CODE
 
+            try:
+                HWR.beamline.lims.lims_rest.authenticate(loginID, password)
```

Comment:
It would be very nice if we can avoid using lims_rest, do you really need it ?

Reply:
We are not in a situation right now to go back to soap auth (even less to start using pyispyb). Also, we rely on the lims_rest for some interactions with ispyb that are not present in soap.

I dont mind removing it in the future, it would be good to have just one hwobj doint the ispyb interaction, we are simply not ready :(

Reply:
I see, how about if we add a login object directly on `HWR.beamline` i.e `HWR.beamline.authentication` which has one maybe two methods `login` and `logout`. Then this can be configured to be for instance lims-rest, ldap or something else ?

It would probably clean up the code abit ? I'm otherwise happy to merge this if you need it

Reply:
Ah sorry, no...

Of course the last required change is to update `pyproject.toml` to fix the lower bound on the version string of the dependency on `mxcubecore`. So it stays as draft for now :D

Reply:
Ok, Ill try to look into how we can generalize the login method and delegate it to a `authentication` hardware object then eventually create a ticket.